src/app/components/forms/register-form/register-form.component.scss

```diff
@@ -0,0 +1,20 @@
+@import '../../../styles/colors';
+
+.register-form-button-container {
+  margin-top: 20px;
+  padding-bottom: 1.5em;
+  text-align: center;
+
+  .register-form-button {
+    margin: 15px auto 30px;
```

Comment:
Properties should be ordered min-width, margin

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)